src/WinRT.Runtime/IWinRTObject.net5.cs

```diff
@@ -167,7 +167,21 @@ internal sealed bool IsInterfaceImplementedFallback(RuntimeTypeHandle interfaceT
                 return false;
             }
 
-            Type helperType = type.FindHelperType();
+            Type helperType;
+            try
+            {
+                helperType = type.FindHelperType();
+            }
+            catch (Exception)

```

Comment:
Wait, when would this throw? We're never catching exceptions from `FindHelperType` elsewhere 🤔

Reply:
This throws if we try to instantiate a custom mapped generic type like a collection type that we haven't registered already in our lookup table.  In most scenarios during Init, we would have already added it.  But this is an is operator scenario, so it isn't added.